java-generator/core/src/main/java/io/fabric8/java/generator/nodes/JObject.java

```diff
@@ -231,6 +231,14 @@ public GeneratorResult generateJava() {
           fieldSetter.addAnnotation(new SingleMemberAnnotationExpr(
               new Name("com.fasterxml.jackson.annotation.JsonFormat"),
               new NameExpr("pattern = \"" + config.getDeserDatetimeFormat() + "\"")));
+          objField.addAnnotation(
+              new SingleMemberAnnotationExpr(
+                  new Name("io.fabric8.crd.generator.annotation.SchemaFrom"),
```

Comment:
I would like to avoid engineering workarounds if possible

Reply:
@andreaTP

With respect to https://github.com/fabric8io/kubernetes-client/issues/5866 (bug in crd-generator) I agree this appears to be a workaround.

However, with respect to https://github.com/fabric8io/kubernetes-client/issues/5867 (crd-generator doesn't understand date-format) I don't consider this a workaround at all.